catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ScoreCommand.java

```diff
@@ -0,0 +1,253 @@
+/**
```

Comment:
A lot of the logic and flow of this command look very similar to the ValidateCommand.java.  We should either add the scoring as an option to the ValidateCommand or abstract the common functionality to avoid duplication.